### ISSUE TITLE
Monk's tokenbar disappeared

### ISSUE THREAD

**Issue by MarceloCortezz** [ironmonk88/monks-tokenbar#28]
I dont know if this can be the cause but when I change my Chrome to Full Screen and dock it next to the macro bar, when I change it back to windowed, Tokenbar doesn't follow the window borders, making it out of the window and unreachable. Maybe this happened because the module itself doesnt show any error.
I've reinstalled, reactivated, deactivated every module but TokenBar and I still have no clue what happened. The only clue I would have is the one I described above.

**Comment by MarceloCortezz** [comment]
UPDATE:
I searched TokenBar through the console and I found it. It was far below my browser window. I don't know much if this is possible but the **element.style.top** of **id=tokenbar** was around 900px.
![image](https://user-images.githubusercontent.com/79060716/108303083-01844700-7184-11eb-9db1-38c237930d04.png)
![image](https://user-images.githubusercontent.com/79060716/108303126-17920780-7184-11eb-881c-d336a502348d.png)
I changed it to around 500 and the bar appeared in my screen. I believe that the module not having a limiter or screen resolution max-positioning can cause this issue.
![image](https://user-images.githubusercontent.com/79060716/108303183-38f2f380-7184-11eb-9701-5040ed439a02.png)
![image](https://user-images.githubusercontent.com/79060716/108303226-47d9a600-7184-11eb-80bc-5113fb4c135c.png)

I found this out using my Foundry (theForge hosted) in two different screens.

**Comment by MarceloCortezz** [comment]
So, basically, having a position limited by screen resolution or snap to bottom screen would be viable solutions.

**Comment by ironmonk88** [comment]
Thank you for the catch.  It's being worked on.  Someone else had a weird repositioning issue and I realized that it was using fixed position related to the top of the screen.  I've changed the code so that if the tokenbar is re positioned near the bottom half of the screen it will position relative to the bottom of the screen.  So when the screen is resized it will remain relative to the bottom edge.  This should alleviate this issue.

Also added a button in the settings that will reset the position so that it's relative to the HotBar.

If you change the value so that it's back on the screen and then drag it somewhere, the new value will be stored and it should stay in the position.

Expect the fix in the next release.
